wasm/memory.cpp

```diff
@@ -74,6 +74,7 @@ extern "C"
 		return dst;
 	}
 
+#ifndef OVERRIDE_MEMSET
```

Comment:
I am skeptical about this. This code explicitly support unaligned accesses which are not otherwise usable. Please try to figure out what is the actual reason musl seems faster.

Reply:
Will do.
Probably it has to do with the fact that memset is in most cases invoked on aligned addresses, and somehow musl's memset code generates better (somehow randomly).
Will remove from this PR, and do a proper benchmark once the rest is in.